financial-connections/src/main/java/com/stripe/android/financialconnections/di/FinancialConnectionsSheetNativeModule.kt

```diff
@@ -53,9 +54,8 @@ internal class FinancialConnectionsSheetNativeModule {
     @Provides
     fun providesNavigationManager(
         logger: Logger
-    ) = NavigationManager(
-        logger = logger,
-        externalScope = CoroutineScope(SupervisorJob() + Dispatchers.Default)
+    ): NavigationManager = NavigationManagerImpl(
```

Comment:
Nit: Could this use a `@Binds` now?

Reply:
https://github.com/stripe/stripe-android/pull/6968/commits/d51d38be17ebd1f551a41ba9d1cd5ca0f3869d4c